src/d3a/models/area/__init__.py

```diff
@@ -388,13 +390,16 @@ def _consume_commands_from_aggregator(self):
     def tick(self):
         self._consume_commands_from_aggregator()
 
-        if ConstSettings.IAASettings.MARKET_TYPE == 2 or \
-                ConstSettings.IAASettings.MARKET_TYPE == 3:
+        if ConstSettings.IAASettings.MARKET_TYPE in [2, 3]:
             if ConstSettings.GeneralSettings.EVENT_DISPATCHING_VIA_REDIS:
                 self.dispatcher.publish_market_clearing()
             else:
                 for market in self.all_markets:
                     market.match_offers_bids()
+        elif ConstSettings.IAASettings.MARKET_TYPE == 4:
+            if self.next_market is not None:
+                self.matcher.calculate_recommendation(*self.next_market.get_open_stock)
+                self.next_market.match_recommendation(self.matcher.bid_offer_pairs)
```

Comment:
Similar to a comment below, we can make this behavior default and not create a separate market type for this.